### PR TITLE
feat: parse references property of manuscript

### PR DESCRIPTION
**Pull request by ilya-zhuravel**

### CODE REVIEW

File: src/model/reference/index.ts [hunk 1]
```diff
@@ -86,3 +86,9 @@ export class Reference extends BackmatterEntity {
     return new refInfoClass(data);
   }
 }
+
+export function createReferencesState(referencesXml: Element[]): Reference[] {
```
Comment (Cooryd) [inline src/model/reference/index.ts:90]: Need a unit test for empty array and array of Elements (can probably copy from one of the other manuscript property state factories).

**Review by Cooryd — APPROVED** [review]
